metastore/scripts/upgrade/hive/upgrade-3.1.0-to-4.0.0.hive.sql

```diff
@@ -302,12 +318,16 @@ SELECT
     WHEN CC_STATE = 'a' THEN 'did not initiate' ELSE 'UNKNOWN' END,
   CASE WHEN CC_WORKER_ID IS NULL THEN cast (null as string) ELSE split(CC_WORKER_ID,"-")[0] END,
   CASE WHEN CC_WORKER_ID IS NULL THEN cast (null as string) ELSE split(CC_WORKER_ID,"-")[1] END,
```

Comment:
Shouldn't this be the same as in hive-schema-4.0.0.hive.sql? i.e. split(CQ_INITIATOR_ID,"-")[size(split(CQ_INITIATOR_ID,"-"))-1]

Reply:
fixed